forui/lib/src/widgets/button/button_content.dart

```diff
@@ -37,6 +37,22 @@ final class _FButtonContent extends StatelessWidget {
   }
 }
 
+final class _FButtonIconContent extends StatelessWidget {
+  final Widget icon;
+
+  const _FButtonIconContent({required this.icon});
+
+  @override
+  Widget build(BuildContext context) {
+    final (:style, enabled: _) = FButton._of(context);
```

Comment:
I think we should introduce a new `FButtonIconStyle`. In general, each content should have its own style.

Reply:
The properties be all same as `FButtonCustomStyle`?

Reply:
I think it's more similar to `FButtonContentStyle`.

Reply:
What should be in the inherit method parameters? The icon content style would be only containing padding (and not text styles as in `FButtonContentStyle`), so should the padding be directly given in the inherit method like

`padding = const EdgeInsets.symmetric(
          horizontal: 16,
          vertical: 12.5,
        )`

or should it be retrieved from any theme property?

Reply:
Yes that is correct, I'm asking in the `inherit` method in the `FButtonIconContentStyle`, what should be the parameters? If the padding be hard-coded as in `FButtonContentStyle` then it wouldn't require any method parameters.

Reply:
I think there isn't a need for a `FButtonIconContentStyle.inherit(...)` constructor. A normal constructor that accepts a padding, with a default value, should suffice.

Reply:
Is it fine now?